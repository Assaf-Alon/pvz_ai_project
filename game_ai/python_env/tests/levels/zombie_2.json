{
    "50": [["Normal", 0], ["Normal", 1], ["Normal", 2], ["Normal", 3], ["Normal", 4]]
}
