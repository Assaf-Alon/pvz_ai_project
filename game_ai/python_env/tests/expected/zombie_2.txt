[100] Sun autospawned. Total: 75.
[200] Sun autospawned. Total: 100.
[300] Sun autospawned. Total: 125.
[400] Sun autospawned. Total: 150.
[500] Normal zombie being spawned at lane 0
[500] Normal zombie being spawned at lane 1
[500] Normal zombie being spawned at lane 2
[500] Normal zombie being spawned at lane 3
[500] Normal zombie being spawned at lane 4
[500] Sun autospawned. Total: 175.
[547] Normal zombie in (0, 9) moved to (0, 8).
[547] Normal zombie in (1, 9) moved to (1, 8).
[547] Normal zombie in (2, 9) moved to (2, 8).
[547] Normal zombie in (3, 9) moved to (3, 8).
[547] Normal zombie in (4, 9) moved to (4, 8).
[594] Normal zombie in (0, 8) moved to (0, 7).
[594] Normal zombie in (1, 8) moved to (1, 7).
[594] Normal zombie in (2, 8) moved to (2, 7).
[594] Normal zombie in (3, 8) moved to (3, 7).
[594] Normal zombie in (4, 8) moved to (4, 7).
[600] Sun autospawned. Total: 200.
[641] Normal zombie in (0, 7) moved to (0, 6).
[641] Normal zombie in (1, 7) moved to (1, 6).
[641] Normal zombie in (2, 7) moved to (2, 6).
[641] Normal zombie in (3, 7) moved to (3, 6).
[641] Normal zombie in (4, 7) moved to (4, 6).
[688] Normal zombie in (0, 6) moved to (0, 5).
[688] Normal zombie in (1, 6) moved to (1, 5).
[688] Normal zombie in (2, 6) moved to (2, 5).
[688] Normal zombie in (3, 6) moved to (3, 5).
[688] Normal zombie in (4, 6) moved to (4, 5).
[700] Sun autospawned. Total: 225.
[735] Normal zombie in (0, 5) moved to (0, 4).
[735] Normal zombie in (1, 5) moved to (1, 4).
[735] Normal zombie in (2, 5) moved to (2, 4).
[735] Normal zombie in (3, 5) moved to (3, 4).
[735] Normal zombie in (4, 5) moved to (4, 4).
[782] Normal zombie in (0, 4) moved to (0, 3).
[782] Normal zombie in (1, 4) moved to (1, 3).
[782] Normal zombie in (2, 4) moved to (2, 3).
[782] Normal zombie in (3, 4) moved to (3, 3).
[782] Normal zombie in (4, 4) moved to (4, 3).
[800] Sun autospawned. Total: 250.
[829] Normal zombie in (0, 3) moved to (0, 2).
[829] Normal zombie in (1, 3) moved to (1, 2).
[829] Normal zombie in (2, 3) moved to (2, 2).
[829] Normal zombie in (3, 3) moved to (3, 2).
[829] Normal zombie in (4, 3) moved to (4, 2).
[876] Normal zombie in (0, 2) moved to (0, 1).
[876] Normal zombie in (1, 2) moved to (1, 1).
[876] Normal zombie in (2, 2) moved to (2, 1).
[876] Normal zombie in (3, 2) moved to (3, 1).
[876] Normal zombie in (4, 2) moved to (4, 1).
[900] Sun autospawned. Total: 275.
[923] Normal zombie in (0, 1) moved to (0, 0).
[923] Normal zombie in (1, 1) moved to (1, 0).
[923] Normal zombie in (2, 1) moved to (2, 0).
[923] Normal zombie in (3, 1) moved to (3, 0).
[923] Normal zombie in (4, 1) moved to (4, 0).
[970] Normal zombie in (0, 0) attempted entering the house.
[970] Normal zombie in (1, 0) attempted entering the house.
[970] Normal zombie in (2, 0) attempted entering the house.
[970] Normal zombie in (3, 0) attempted entering the house.
[970] Normal zombie in (4, 0) attempted entering the house.
[970] Normal zombie in (0, 0) triggered a lawnmower.
[970] Lawnmower in (0, 0) attacked.
[970] Normal zombie in (0, 0) was damaged. HP: -9818.
[970] Normal zombie in (0, 0) was killed.
[970] Normal zombie in (1, 0) triggered a lawnmower.
[970] Lawnmower in (1, 0) attacked.
[970] Normal zombie in (1, 0) was damaged. HP: -9818.
[970] Normal zombie in (1, 0) was killed.
[970] Normal zombie in (2, 0) triggered a lawnmower.
[970] Lawnmower in (2, 0) attacked.
[970] Normal zombie in (2, 0) was damaged. HP: -9818.
[970] Normal zombie in (2, 0) was killed.
[970] Normal zombie in (3, 0) triggered a lawnmower.
[970] Lawnmower in (3, 0) attacked.
[970] Normal zombie in (3, 0) was damaged. HP: -9818.
[970] Normal zombie in (3, 0) was killed.
[970] Normal zombie in (4, 0) triggered a lawnmower.
[970] Lawnmower in (4, 0) attacked.
[970] Normal zombie in (4, 0) was damaged. HP: -9818.
[970] Normal zombie in (4, 0) was killed.
[970] You've won!
